tools/conform-includes.js

```diff
@@ -0,0 +1,89 @@
+const fs = require("fs");
```

Comment:
Nice! Would be good to have a comment at the top explaining what this is and how to use it.